### ISSUE TITLE
Bookmarking Podcast

### ISSUE THREAD

**Issue by bensmith2697** [CappielloAntonio/tempo#79]
Do you have plans to add bookmarking to podcasts? Currently, the podcast always starts from the beginning but it would be nice for it start from where I left off. DSub already does this but I really prefer the interface of this application.

**Comment by bensmith2697** [comment]
@CappielloAntonio could you please respond to the above?

**Comment by CappielloAntonio** [comment]
Yes, it's in the plans but I can't give you a date as there are bugs to be fixed and the time to work on them is limited

**Comment by bensmith2697** [comment]
@CappielloAntonio my apologies, I will keep a lookout for this then (hopefully soon, I really love this app) and use DSub in the meantime. Thank you for the response.
